pallets/nft/src/tests.rs

```diff
@@ -3,159 +3,732 @@ use crate::mock::{Event, *};
 use frame_support::{assert_noop, assert_ok};
 use sp_runtime::AccountId32;
 
+fn initial_account() -> (AccountId32, AccountId32) {
+	let alice_account: AccountId32 = AccountId32::from([
+		0xd4, 0x35, 0x93, 0xc7, 0x15, 0xfd, 0xd3, 0x1c, 0x61, 0x14, 0x1a, 0xbd, 0x04, 0xa9, 0x9f,
+		0xd6, 0x82, 0x2c, 0x85, 0x58, 0x85, 0x4c, 0xcd, 0xe3, 0x9a, 0x56, 0x84, 0xe7, 0xa5, 0x6d,
+		0xa2, 0x7d,
+	]);
+
+	let bob_account: AccountId32 = AccountId32::from([
```

Comment:
For alice, it's understandable, but why we need this complex bob?

Reply:
Typically we need two accounts for test transfer etc. This is for claim test, a merkleroot sample account.

Reply:
I mean, why not use 0:32

Reply:
Also, pls do not resolve issues by yourself

Reply:
ok... Bad habbit. I will the issue unresolved next time for your reference